manifests/g/gaphor/gaphor/2.18.1/gaphor.gaphor.locale.en-US.yaml

```
@@ -0,0 +1,71 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: gaphor.gaphor
+PackageVersion: 2.18.1
+PackageLocale: en-US
+Publisher: gaphor
```

```suggestion
Publisher: Gaphor Community
```
This must match Control Panel.